branding/ghostery/pref/firefox-branding.js

```diff
@@ -99,3 +99,6 @@ pref("browser.crashReports.unsubmittedCheck.autoSubmit2", false); // [FF58+]
 pref("browser.messaging-system.whatsNewPanel.enabled", false); // What's New [FF69+]
 pref("extensions.pocket.enabled", false); // Pocket Account [FF46+]
 pref("identity.fxaccounts.enabled", false); // Firefox Accounts & Sync [FF60+] [RESTART]
+
+// UA compat mode - Adds Firefox/VER to the UA string in addition to the APP_NAME.
```

Comment:
Maybe link to original issue to make sure we can track context in the future?
```suggestion
// UA compat mode - Adds Firefox/VER to the UA string in addition to the APP_NAME. (https://github.com/ghostery/user-agent-desktop/issues/114)
```